pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -42,9 +42,10 @@ const (
 )
 
 var (
-	// The kernel has to be newer than 4.7.0 since we are using bpf_skb_load_bytes (4.5.0+) method to read from the
-	// socket filter, and a tracepoint (4.7.0+).
-	classificationMinimumKernel = kernel.VersionCode(4, 7, 0)
+	// The kernel has to be newer than 4.11.0 since we are using bpf_skb_load_bytes (4.5.0+), which
```

Comment:
Does this mean protocol classification never worked for < 4.11?

Reply:
Yeah. `system-probe` fails to run on debian 9 unless I disable protocol classification.